docs/framework/windows-workflow-foundation/samples/basic-validation.md

```diff
@@ -23,7 +23,7 @@ This sample consists of an activity, `CreateProduct`, which validates that its `
   
 -   Create a constraint (`PriceGreaterThanCost`). This is where all the validation logic resides.  
   
--   Override <xref:System.Activities.CodeActivity%601.OnGetConstraints%2A> and add the constraint (`PriceGreaterThanCost`) to the constraints <xref:System.Collections.IList>.  
+-   Override `System.Activities.CodeActivity.OnGetConstraints()` and add the constraint (`PriceGreaterThanCost`) to the constraints <xref:System.Collections.IList>.  
```

Comment:
AFAICT they updated the samples but not the docs. Yes, it needs more extensive work. This is noted in the Migration project for further action. I'll open an issue on Monday and exchange the note in the Migration Project with the issue I open.